packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.spec.ts

```diff
@@ -461,4 +501,254 @@ describe('PaypalCommerceButtonStrategy', () => {
             }));
         });
     });
+    describe('#_onApprove button callback', () => {
```

Comment:
Please add an empty line above

Reply:
May I ask you to add tests for ShippingOptions callbacks too?

Reply:
added